cms/djangoapps/contentstore/views/__init__.py

```diff
@@ -8,6 +8,7 @@
 from .utility import *
 from .utilities.captions import *
 from .utilities.bulksettings import *
+from openedx.stanford.cms.djangoapps.contentstore.views.utilities.bulkupdate import *
```

Comment:
I _think_ we can remove this line;
I don't believe we're actively using it here, at this point.

Let's give it a shot..

Reply:
This did not work